codegen/src/main/kotlin/tools/samt/codegen/http/HttpTransport.kt

```diff
@@ -32,9 +32,10 @@ object HttpTransportConfigurationParser : TransportConfigurationParser {
                 val serviceName = service.name
                 val operationConfiguration = operationsField.asObject
 
-                val parsedOperations = operationConfiguration.fields
-                    .filterKeys { it.asIdentifier != "basePath" }
-                    .mapNotNull { (key, value) ->
+                val operationConfigurations = operationConfiguration.fields.filterKeys { it.asIdentifier != "basePath" }
+                val parsedOperations = buildList<HttpTransportConfiguration.OperationConfiguration> {
+                    val list = this
```

Comment:
It might be more readable to just skip the `buildList` entirely for this part

```suggestion
                val parsedOperations = mutableListOf<HttpTransportConfiguration.OperationConfiguration>()
```